zwave_js_server/__main__.py

```diff
@@ -62,9 +62,9 @@ def log_value_updated(event: dict) -> None:
         value = event["value"]
 
         if node.device_config:
-            description = node.device_config["description"]
+            description = node.device_config.description
         else:
-            description = f'{node.device_class["generic"]} (missing device config)'
+            description = f'{node.device_class.generic} (missing device config)'
```

Comment:
Should we guard against a None value here on `node.device_class.generic`?

Reply:
should always be present

Reply:
It's typed as Optional in the device class.

Reply:
I'll fix this tomorrow. Merging as is.